src/controllers/connectionManager.ts

```diff
@@ -600,6 +601,35 @@ export default class ConnectionManager {
             // close active connection
             await this.disconnect(fileUri);
             // connect to the server/database
+            if (connectionCreds.authenticationType === Constants.azureMfa) {
+                if (!connectionCreds.azureAccountToken) {
+                    let account = this.accountStore.getAccount(connectionCreds.accountId);
+                    let profile = new ConnectionProfile();
+                    profile.accountId = connectionCreds.accountId;
```

Comment:
It would be cleaner if we could just add a static method to create connection profiles in one constructor/helper function in the connection profile class